src/main/java/com/gmail/val59000mc/scenarios/scenariolisteners/GoneFishingListener.java

```diff
@@ -1,30 +1,51 @@
 package com.gmail.val59000mc.scenarios.scenariolisteners;
 
+import com.gmail.val59000mc.UhcCore;
 import com.gmail.val59000mc.events.UhcStartedEvent;
 import com.gmail.val59000mc.exceptions.UhcPlayerNotOnlineException;
 import com.gmail.val59000mc.players.UhcPlayer;
 import com.gmail.val59000mc.scenarios.ScenarioListener;
+import org.bukkit.ChatColor;
 import org.bukkit.Material;
 import org.bukkit.enchantments.Enchantment;
 import org.bukkit.event.EventHandler;
 import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
 
 public class GoneFishingListener extends ScenarioListener{
 
     @EventHandler
     public void onGameStarted(UhcStartedEvent e){
         ItemStack rod = new ItemStack(Material.FISHING_ROD);
-        rod.addEnchantment(Enchantment.LURE, 3);
-        rod.addEnchantment(Enchantment.LUCK, 3);
-        rod.addEnchantment(Enchantment.DURABILITY, 3);
+        rod.addUnsafeEnchantment(Enchantment.LURE, 8);
+        rod.addUnsafeEnchantment(Enchantment.LUCK, 255);
+
+        ItemMeta meta = rod.getItemMeta();
+        // Set item unbreakable code change in 1.15 and older versions
+
+        if(UhcCore.getVersion() < 15){
+            meta.spigot().setUnbreakable(true);
+        }else{
+            meta.setUnbreakable(true);
+        }
```

Comment:
Please move version specific code to VersionUtils.

Reply:
I am sorry but I don't know how I can do that, it is my first collaboration to open source project...

Reply:
Alright, than dw. I'll change that part.

Reply:
Will the code be added to the plugin?